First draft of article (#1)
arfc/2021-bachmann-epjn: results.tex

\section{Results}
The results presented for each scenario include the energy produced, reactor 
deployment schedule, enriched
uranium mass, \gls{HALEU} mass, and \gls{SWU} capacity required as a function of time. 

\subsection{Scenario 1}
The amount of energy supplied by the \glspl{LWR} and the number of \glspl{LWR}

> deployed as a function of time is shown in Figure \ref{fig:energy_rx_1}. 

Reviewer: ```suggestion
deployed as a function of time are shown in Figure \ref{fig:energy_rx_1}. 
```

\glspl{LWR} are first deployed in August of 1967, and the last 
\gls{LWR} is deployed in June of 2016 and decommissioned in July 2076. The 
maximum number of 
\glspl{LWR} deployed at one time is 109. The energy produced by the 

> \glspl{LWR} follows with the number of reactors deployed. The maximum energy 

Reviewer: ```suggestion
\glspl{LWR} follows the number of reactors deployed. The maximum energy 
```

\begin{figure}
    \centering 
    \includegraphics[width=\textwidth]{figures/fuelsupply_scenarios_1.png}
    \caption{Mass of enriched uranium sent to reactors in Scenario 1.}
    \label{fig:fuel_1}
\end{figure}

Finally, the \gls{SWU} capacity to produce fuel for the scenario is shown in 
Figure \ref{fig:swu_1}. The \gls{SWU} capacity required as a function of 
time follows the mass of uranium sent to the reactors, since the \gls{SWU}
is calculated based on those transactions. An average of 0.74$\times 10^6$
kg-\gls{SWU}/month and a maximum of 3.95$\times 10^6$ kg-\gls{SWU} is 
required for this scenario. Prior to 2025, an average of 1.21$\times 10^6$ 
kg-\gls{SWU}/month is required to enrich the uranium sent to the \glspl{LWR}.

\begin{figure}
    \centering
    \includegraphics[width=\textwidth]{figures/totalswu_scenarios_1.png}
    \caption{Total \gls{SWU} capacity required to produce fuel sent to the 
    reactors at each time step in Scenario 1.}
    \label{fig:swu_1}
\end{figure}

The deployment and decommissioning schedule of the \glspl{LWR} in this 
scenario is 
applied to each of the other scenarios, and therefore the material 

> requirements of each of the following scenarios is the same as those 

Reviewer: ```suggestion
requirements of each of the following scenarios are the same as those 
```

of this scenario prior to 2025. 

\subsection{Scenario 2}
The energy produced by each type of reactor, the transition energy demand, 
and the deployment schedule of the \glspl{MMR} for Scenario 2 are shown in 
Figure \ref{fig:energy_rx_2}. The first \glspl{MMR} are deployed in 
October 2031, and the 
maximum number deployed at one time is 
9,182. 

Once the transition begins in 2025, there are some time periods in which 
the energy demand of the scenario is 
not met. The first of these is between 2030-2050, with a maximum deficit 
of 5.7855 GWe-y in 2032. Other periods in which the energy demand is not met 
correspond to the decommissioning of \glspl{MMR} at the end of their lifetime 
and the deployment of new reactors, such as from 2062-2069. 
The deployment of \glspl{MMR} in 2031 contributes to the inability to 

> meet the energy demand of the scenario between 203-2050, because the energy 

Reviewer: ```suggestion
meet the energy demand of the scenario between 2030-2050, because the energy 
```

    \label{fig:energy_rx_2}
\end{figure}

Next, Figure \ref{fig:fuel_2} shows the mass of enriched uranium sent to all 
reactors in the scenario and just to the \glspl{MMR}. The \glspl{MMR} 
do not require refueling during their lifetime, so uranium is only 
sent to these reactors when they are deployed. This causes the 
large increases in uranium sent to these reactors and the periods of 
no uranium sent to them. An average of 104.94 MTU/month and a maximum 
of 781.41 MTU is sent to all of the reactors in the scenario after 2025. 
This average is lower than the average mass of enriched uranium 
sent to the \glspl{LWR} in Scenario 1, but the maximum exceeds the maximum 
mass of enriched uranium sent to \glspl{LWR} in Scenario 1 by 267.71 MT.
An average of 48.50 MTU/month and a maximum of 719.25 
MTU is sent to the \glspl{MMR} once they are deployed in 2031. The average 
mass of enriched uranium sent to the 
\glspl{MMR} is lower than the average mass of uranium sent to the \glspl{LWR}
prior to 2025. This is despite a much higher maximum and multiple peaks higher
than
average mass sent to the \glspl{LWR}. The lower average is explained by the 
multiple time periods in 

> which no \gls{HALEU} is sent to the \glspl{MMR} due to the lack of refuleing.  

Reviewer: ```suggestion
which no \gls{HALEU} is sent to the \glspl{MMR} due to the lack of refueling.  
```

\begin{figure}
    \centering
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/fuelsupply_scenarios_2.png}
        \caption{Mass of enriched uranium sent to all reactors.}
        \label{fig:totalfuel_2}
    \end{subfigure}
    \hspace{0.8cm}
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/advancedRX_fuelsupply_scenarios_2.png}
        \caption{Mass of enriched uranium sent to \glspl{MMR}.}
        \label{fig:haleu_2}
    \end{subfigure}
    \caption{Uranium mass sent to reactors in Scenario 2.}
    \label{fig:fuel_2}
\end{figure}

Figure \ref{fig:swu_2} shows the \gls{SWU} capacity needed to 
enrich uranium for all reactors in the scenario, and to enrich uranium for 
just the \glspl{MMR}. There is a large difference in the \gls{SWU} 
capacity needed to enrich uranium for \glspl{LWR} and \glspl{MMR}. This 
is because the \glspl{MMR} require a much larger mass and 

> a higher enrchment level. An average of 1.37$\times 10^6$ kg-\gls{SWU}/month

Reviewer: ```suggestion
a higher enrichment level. An average of 1.37$\times 10^6$ kg-\gls{SWU}/month
```

and a maximum of 20.3$\times 10^6$ kg-\gls{SWU}

> is needed to enrich the uranium sent to \glspl{MMR}. These values are both 

Reviewer: ```suggestion
are needed to enrich the uranium sent to \glspl{MMR}. These values are both 
```

higher than the 
average and maximum \gls{SWU} capacity needed to enrich fuel for the 
\glspl{LWR} in Scenario 1. 

\begin{figure}
    \centering
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/totalswu_scenarios_2.png}
        \caption{Total \gls{SWU} required to enrich uranium sent to all reactors at each time step.}
        \label{fig:totalswu_2}
    \end{subfigure}
    \hspace{0.8cm}
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/haleuSWU_scenarios_2.png}
        \caption{\gls{SWU} required to enrich uranium sent to \glspl{MMR} at each time step.}
        \label{fig:haleuswu_2}
    \end{subfigure}
    \caption{\gls{SWU} required to enrich natural uranium in Scenario 2.}
    \label{fig:swu_2}
\end{figure}

\subsection{Scenario 3}
For Scenario 3, Figure \ref{fig:energy_rx_3} shows the number of Xe-100 reactors deployed, 
the energy produced by each reactor type, and the energy demand. 
Xe-100 reactors are deployed in October 2031, which is the same as when 
\glspl{MMR}

> are deployed in Scenario 2. A maximum of 1225 Xe-100 reactors are deployed.

Reviewer: ```suggestion
are deployed in Scenario 2. Scenario 3 deploys a maximum of 1,225 Xe-100 reactors.
```

\gls{SWU} capacity needed to enrich uranium between Scenarios 1 and 3, 
despite the higher enrichment level of uranium 
sent to the Xe-100 reactors. This is because the Xe-100 reactors receive 
a lower average mass of enriched uranium at each time step. Enriching uranium 
for the Xe-100 reactors requires an average of 
0.82$\times 10^6$ kg-\gls{SWU}/month and a maximum of 3.64$\times 10^6$
kg-\gls{SWU}.  

\begin{figure}
    \centering
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/totalswu_scenarios_3.png}
        \caption{\gls{SWU} required to enrich uranium sent to all reactors.}
        \label{fig:totalswu_3}
    \end{subfigure}
    \hspace{0.8cm}
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/haleuSWU_scenarios_3.png}
        \caption{\gls{SWU} required to enrich uranium sent to Xe-100s.}
        \label{fig:haleuswu_3}
    \end{subfigure}
    \caption{\gls{SWU} required to enrich natural uranium in Scenario 3.}
    \label{fig:swu_3}
\end{figure}

\subsection{Scenario 4}
Figure \ref{fig:energy_rx_4} shows the number of \glspl{MMR} deployed, 

> energy produced by each type of reactor, and the energy demand of this

Reviewer: ```suggestion
the energy produced by each type of reactor, and the energy demand of this
```

capacity required to enrich uranium for the \glspl{MMR} in 
Scenario 2. The average is slightly higher than the average \gls{SWU} 
capacity required to enrich uranium for the \glspl{LWR} before 2025 in 
Scenario 1. 

\begin{figure}
    \centering
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/totalswu_scenarios_4.png}
        \caption{\gls{SWU} required to enrich uranium sent to all reactors.}
        \label{fig:totalswu_4}
    \end{subfigure}
    \hspace{0.8cm}
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/haleuSWU_scenarios_4.png}
        \caption{\gls{SWU} required to enrich uranium sent to \glspl{MMR}.}
        \label{fig:haleuswu_4}
    \end{subfigure}
    \caption{\gls{SWU} required to enrich natural uranium in Scenario 4.}
    \label{fig:swu_4}
\end{figure}

\subsection{Scenario 5}
Figure \ref{fig:energy_rx_5} shows the number of Xe-100 reactors, the 
energy produced by each type of reactor, and the energy demand of the 
scenario. The Xe-100 reactors are deployed in January 2030, the 
same as when 
\glspl{MMR} are deployed in Scenario 4. The maximum number of Xe-100 

> reactors deployed in the scenario is 2339. 

Reviewer: ```suggestion
reactors deployed in the scenario is 2,339. 
```

values are both lower than the uranium mass sent to the 
\glspl{LWR} in Scenario 1. An average of 
38.11 MTU/month and a maximum of 117.98 MTU is sent to the Xe-100 reactors. 
These metrics are all much lower than what is observed 
for fueling the \glspl{MMR} in Scenario 4.

\begin{figure}
    \centering
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/fuelsupply_scenarios_5.png}
        \caption{Mass of enriche uranium sent to all reactors.}
        \label{fig:totalfuel_5}
    \end{subfigure}
    \hspace{0.8cm}
    \begin{subfigure}{0.5\textwidth}
        \centering
        \includegraphics[scale=0.5]{figures/advancedRX_fuelsupply_scenarios_5.png}
        \caption{Mass of enriched uranium sent to Xe-100s.}
        \label{fig:haleu_5}
    \end{subfigure}
    \caption{Uranium mass sent to reactors in Scenario 5.}
    \label{fig:fuel_5}
\end{figure}

Finally, Figure \ref{fig:swu_5} shows the \gls{SWU} capacity required
to enrich the uranium sent to all of the reactors, and just the Xe-100
reactors. The \gls{SWU} capacity needed 
to enrich uranium for the Xe-100 reactors starts out at a similar 
amount as the \glspl{LWR}, but 

> increases as the enrgy demand and the number of reactors increases. 

Reviewer: ```suggestion
increases as the energy demand and the number of reactors increases. 
```
